processor/transform/readme-components.ts

```diff
@@ -64,6 +64,26 @@ const coerceJsxToMd =
         },
       };
 
+      parent.children[index] = mdNode;
+    } else if (node.name === 'HTMLBlock') {
+      const { position } = node;
+      const children = getChildren<HTMLBlock['children']>(node);
```

Comment:
Is there ever more than one child? It seems weird to me to support multiple children:

```
<HTMLBlock>
  {'<span>Hi</span>'}
  {'<span>Hello!</span>'}
</HTMLBlock>
```

Reply:
oh, no, it only ever expects one child, but i didn't want to make that util unnecessarily narrow

Reply:
Fair enoudh.